onadata/apps/api/tasks.py

```diff
@@ -101,9 +101,9 @@ def send_verification_email(email, message_txt, subject):
     """
     Sends a verification email
     """
-    send_generic_email(email, subject, message_txt)
+    send_generic_email(email, message_txt, subject)
 
 
 @task()
-def send_account_lockout_email(email, subject, message_txt):
-    send_generic_email(email, subject, message_txt)
+def send_account_lockout_email(email, message_txt, subject):
```

Comment:
I suppose you will also need to change everywhere `send_account_lockout_email()` has been used e.g. in .`onadata/libs/authentication.py`